### DISCUSSION TITLE
计算圆周率 π | boringhex.top 博客

### DISCUSSION THREAD

**Discussion by giscus** [boringhex-top/blog.boringhex.top#10] [Announcements]
# 计算圆周率 π | boringhex.top 博客

今天在电脑上尝试计算圆周率，把代码和用时贴出来，有兴趣可以在评论区写出你的用时。

https://blog.boringhex.top/posts/7cb404ab7ccb/

**Comment by pengwon** [comment]
GitHub上开源的一本数学书《数学元素》就以这个问题为一个线索展开，很有意思。
